shell/common/engine.h

```diff
@@ -363,6 +363,13 @@ class Engine final : public RuntimeDelegate, PointerDataDispatcher::Delegate {
   ///
   [[nodiscard]] bool Restart(RunConfiguration configuration);
 
+  //----------------------------------------------------------------------------
+  /// @brief      Setup default font manager according to specific platform.
+  ///
+  /// @attention  This operation calls `SkFontMgr::RefDefault` which is
```

Comment:
Remove the `@attention` comment.  The call to `SkFontMgr::RefDefault` is an implementation detail that will vary across platforms.

Reply:
done